### ISSUE TITLE
Can't start Thunderbird in KDE any more

### ISSUE THREAD

**Issue by curiosityseeker** [roddhjav/apparmor.d#171]
I have my own thunderbird profile which looks like this:

`profile thunderbird /usr/lib/thunderbird/thunderbird{,-bin} { `

When starting it in KDE Plasma I get the following error:

`ALLOWED plasmashell exec /usr/bin/thunderbird info="profile transition not found" comm=plasmashell requested_mask=x denied_mask=x class=file error=-13`

When commenting `include <abstractions/app-launcher-user>`in the plasmashell profile this error doesn't happen. Another workaround is to change the corresponding lines in the app-launcher-user abstraction:

```
# Thunderbird
  /{usr/,}bin/thunderbird{,.sh,-esr,-bin}                                        rPx -> thunderbird,
  /{usr/,}lib{,32,64}/thunderbird{,.sh,-esr,-bin}/thunderbird{,.sh,-esr,-bin}    rPx -> thunderbird,
  /opt/thunderbird{,.sh,-esr,-bin}/thunderbird{,.sh,-esr,-bin}                   rPx -> thunderbird,
```
I'm not sure why this is necessary here.

**Comment by roddhjav** [comment]
Tldr: The attachments for the Thunderbird profile defined in the profiles and the one in the abstraction should be the same, and you need all of them.

plasmashell wants to start `/usr/bin/thunderbird`, but it cannot because there is not profile for `/usr/bin/thunderbird`, but for `/usr/lib/thunderbird/thunderbird{,-bin}`

Your alternative solution (using `-> thunderbird`) works, however, it is limited to profile using the abstraction.

**Comment by curiosityseeker** [comment]
Embarrassing - how  obvious! So I changed my profile to 

```
@{tb_path}  = /{usr/,}lib/thunderbird/thunderbird{,-bin}
@{tb_path} += /{usr/,}bin/thunderbird{,-bin}
profile thunderbird @{tb_path} {
```
and all is well without the need to change `app-launcher-user`.
